app/tests/spec/views/mixins/flow-events-mixin.js

```diff
@@ -0,0 +1,154 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define((require, exports, module) => {
+  'use strict';
+
+  const $ = require('jquery');
+  const { assert } = require('chai');
+  const flowEventsMixin = require('views/mixins/flow-events-mixin');
+  const sinon = require('sinon');
+
+  const FLOW_ID = 'F1031DF1031DF1031DF1031DF1031DF1031DF1031DF1031DF1031DF1031DF103';
+
+  describe('views/mixins/flow-events-mixin', () => {
+    it('exports correct interface', () => {
+      assert.lengthOf(Object.keys(flowEventsMixin), 5);
+      assert.isFunction(flowEventsMixin.afterRender);
+      assert.lengthOf(flowEventsMixin.afterRender, 0);
+      assert.notOk(flowEventsMixin.flow);
+      assert.deepEqual(flowEventsMixin.events, {
```

Comment:
There are tests for them in the mixing views, plus the identified methods are tested directly further down in this suite.